compiler/frontend-spark/src/main/java/edu/snu/onyx/compiler/frontend/spark/JavaSparkContext.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.snu.onyx.compiler.frontend.spark;
+
+import java.io.Serializable;
+
+/**
+ * Spark context wrapper for Java.
+ */
+public final class JavaSparkContext {
```

Comment:
extend `org.apache.spark.api.java.JavaSparkContext`.

Reply:
Please refer to the comment I've left below.